components/button/test/button-mixin.test.js

```diff
@@ -24,7 +24,7 @@ describe('ButtonMixin', () => {
 	});
 
 	after(() => {
-		document.removeEventListener('click', documentClickHandler);
+		document.removeEventListener('click', documentClickHandler, {once: true});
```

Comment:
Changed to fix test failure in edge where this event listener is not being removed causing test failures in button tests